libdevcore/Log.h

```diff
@@ -265,7 +262,8 @@ class LogOutputStream: LogOutputStreamBase
 	LogOutputStream(): LogOutputStreamBase(Id::name(), &typeid(Id), Id::verbosity, _AutoSpacing) {}
 
 	/// Destructor. Posts the accrued log entry to the g_logPost function.
-	~LogOutputStream() { if (Id::verbosity <= g_logVerbosity) g_logPost(m_sstr.str(), Id::name()); }
+	~LogOutputStream() { if (Id::verbosity <= g_logVerbosity)
+			debugOut(m_sstr.str()); }
```

Comment:
weird-looking indentation, maybe just move `if` and braces to separate line each